ember-cli-build.js

```diff
@@ -23,10 +23,6 @@ module.exports = function(defaults) {
   }
 
   var app = new EmberApp(defaults, {
-    'ember-cli-babel': {
-      // async, await, etc.
-      includePolyfill: true
-    },
```

Comment:
Turns out we no longer need this. Probably due to ditching phantomjs. We used to need it to make use of `async/await`